doc/troubleshooting/howtoset_num_nodes.md

```diff
@@ -0,0 +1,16 @@
+# How to control the number of nodes used by a job ?
+
+Set the number of CPU nodes used by DP algorithms with:
+```bash
+mpirun -np $num_nodes dp
+```
+Set the number of threads used by DP algorithms with:
+```bash
+export OMP_NUM_THREADS = $num_threads
+```
+
+Set the number of CPU nodes used by TF kernels with:
+```bash
+export TF_INTRA_OP_PARALLELISM_THREADS = $num_nodes
+export TF_INTER_OP_PARALLELISM_THREADS = $num_nodes
```

Comment:
```suggestion
export TF_INTRA_OP_PARALLELISM_THREADS=$num_nodes
export TF_INTER_OP_PARALLELISM_THREADS=$num_nodes
```

Reply:
@tuoping  Please run and test what you have written, make sure that when a user follows your instruction will get the behavior you expected.

Reply:
OK, sorry.